src/App.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import styled from "styled-components";
 import "./App.css";
-import InputPins from "./components/InputPins";
-import Frame from "./components/frame";
+import InputPins from "./components/tsx components/InputPins";
+import Frame from "./components/tsx components/frame";
 
-function totalScore(arr) {
+function totalScore(arr: number[]) {
```

Comment:
We could implement some improvements to this function: 


      /**
       * 1. Converter estrutura flat em estrutura de frames [[4 , 5], [10], [3, 6]]
       * 2. Criar variáveis para valores utilizados em múltiplas condicionais.
       * 3. Criar função parar somar valores de strike e spare:
       * valorAnterior + somarStrike(frames[i + 1], frames[i + 2])
       * valorAnterior + somarSpare(frames[i + 1])
       *
       */